docs/docker-image-details.md

```diff
@@ -96,6 +96,13 @@ You can use these kinds of Docker images to facilitate integration testing of al
 You can use this kind of image to test the catalog of Apache Hive with kerberos enable
 
 Changelog
+- gravitino-ci-kerberos-hive:0.1.5
+  - Start another HMS service in the container with port 19083.
```

Comment:
Please describe the differences between this HMS and another one, as well as their use cases.

Reply:
done